### DISCUSSION TITLE
Event/Site Awards Not Counting on Global

### DISCUSSION THREAD

**Discussion by StingX2** [RetroAchievements/RAWeb#1278] [Ideas]
Before one of the big issues with players playing events was the badges would hurt their Completion %, now we have one obstacle left. On the global leaderboard it includes events and site awards amongst your total masteries.

Would it be possible to just include these two categories on the global leaderboard? The global leaderboards focus is on points and game masteries and not events/dev awards/site awards. This would be the last needed step that keeps some players uninterested in events.

**Comment by Jamiras** [comment]
Fixed in #1912 ([link](https://github.com/RetroAchievements/RAWeb/commit/e13c9841feb324b3da8f8f8ed8f1a40d879ab1f4#diff-22a67ae835f62e2222904dd427d59a726fe8da2cc09c0452388c942217582222R16))
